amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -21,124 +20,186 @@ pub struct Transform {
 
 impl Transform {
     /// Rotate to look at a point in space (without rolling)
-    pub fn look_at(&mut self, orientation: &Orientation, position: Point3<f32>) -> &mut Self {
-        self.rotation = Quaternion::look_at(
-            position - Point3::from_vec(self.translation),
-            orientation.up.into(),
-        ).into();
+    // Does this make sense, because the position doesn't take into account parent transformations?
+    #[inline]
+    pub fn look_at(&mut self, up: Vector3<f32>, position: Point3<f32>) -> &mut Self {
+        self.rotation = Quaternion::look_at(position - Point3::from_vec(self.translation), up);
         self
     }
 
     /// Returns the local object matrix for the transform.
     ///
     /// Combined with the parent's `GlobalTransform` component it gives
     /// the global (or world) matrix for the current entity.
+    // this is a hot function
```

Comment:
First letter uppercase please x)
Also it should either be /// or it should be inside of the function //.